vm/hugepage/libhugetlbfs/dependency.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# For glibc-static dependence issue on rhel8
+# Bug 1567712 - glibc-static needs to be multilib on RHEL 8
+
+this_arch=$(uname -m)
+this_distro=$(grep DISTRO /etc/motd | cut -d '-' -f 3)
```

Comment:
It would be nice to have an example of `/etc/motd` (or at least `$this_distro`) in a comment. Reading this, I have no idea what `/etc/motd` usually contains (my local one is empty!) so it's hard to interpret what the rest of the code does.